packages/generator-langium/langium-template-web/package.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "<%= extension-name %>",
+    "displayName": "<%= extension-name %>",
+    "description": "Please enter a brief description here",
+    "version": "0.0.1",
+    "engines": {
+        "vscode": "^1.67.0"
+    },
+    "categories": [
+        "Programming Languages"
+    ],
+    "contributes": {
+        "languages": [{
+            "id": "<%= language-id %>",
+            "aliases": ["<%= RawLanguageName %>", "<%= language-id %>"],
+            "extensions": <%= file-extension %>,
+            "configuration": "./language-configuration.json"
+        }],
+        "grammars": [{
+            "language": "<%= language-id %>",
+            "scopeName": "source.<%= language-id %>",
+            "path": "./syntaxes/<%= language-id %>.tmLanguage.json"
+        }]
+    },
+    "activationEvents": [
+        "onLanguage:<%= language-id %>"
+    ],
+    "files": [
+        "bin",
+        "out",
+        "src"
+    ],
+    "bin": {
+        "<%= language-id %>-cli": "./bin/cli"
+    },
+    "main": "./out/extension.js",
+    "scripts": {
+        "vscode:prepublish": "npm run build && npm run lint",
+        "build": "tsc -b tsconfig.json",
+        "watch": "tsc -b tsconfig.json --watch",
+        "lint": "eslint src --ext ts",
+        "langium:generate": "langium generate",
+        "langium:watch": "langium generate --watch",
+        "build:worker": "esbuild --minify ./out/language-server/main-browser.js --bundle --format=iife --outfile=./public/<%= language-id %>-server-worker.js",
+        "prepare:public": "npx shx mkdir -p ./public && npx shx cp -fr ./src/static/* ./public/",
+        "copy:monaco-editor-wrapper": "npx shx cp -fr ./node_modules/monaco-editor-wrapper/bundle ./public/monaco-editor-wrapper",
+        "copy:monaco-workers": "npx shx cp -fr ./node_modules/monaco-editor-workers/dist/ ./public/monaco-editor-workers",
+        "build:web": "npm run build && npm run prepare:public && npm run build:worker && npm run copy:monaco-editor-wrapper && npm run copy:monaco-workers",
+        "serve": "node ./out/web/app.js"
+    },
+    "dependencies": {
+        "chalk": "~4.1.2",
+        "chevrotain": "~10.4.2",
+        "commander": "~10.0.0",
+        "express": "~4.18.2",
+        "langium": "~1.1.0",
+        "monaco-editor-workers": "~0.34.2",
+        "monaco-editor-wrapper": "~1.6.0",
+        "vscode-languageclient": "~8.0.2",
```

Comment:
I was just wondering if we shouldn't just rely on the transitive dependencies `langium` provides. These versions will only get out of sync (forgotten to update) and increase maintenance.

Reply:
There is no transitive dependency, since this is the project that implements the language server.

Reply:
Sorry, you are right. For the `vscode-languageclient` there is none, but for `vscode-languageserver` and `vscode-uri` we could rely on the ones defined by langium.

Reply:
What about `chevrotain`, should it also be removed from this package.json?

Reply:
Makes sense to me, too. @msujew WDYT?

Reply:
I guess so, yes.